ParticleObjects/particle_class.f90

```diff
@@ -100,6 +100,10 @@ module particle_class
     real(defReal)              :: w         ! Particle Weight
     real(defReal)              :: time      ! Particle time point
 
+    ! Information passed from geometry
+    real(defReal)              :: T = -INF   ! Local temperature
```

Comment:
You could use a consistent undefined value in between places: it is -INF here but, for example, -ONE in fissionSource_class.f90 (line 227)